substrate/frame/system/src/lib.rs

```diff
@@ -750,7 +753,9 @@ pub type RefCount = u32;
 /// Information of an account.
 #[derive(Clone, Eq, PartialEq, Default, RuntimeDebug, Encode, Decode, TypeInfo, MaxEncodedLen)]
 pub struct AccountInfo<Nonce, AccountData> {
-	/// The number of transactions this account has sent.
+	/// A value that increases with every transaction that the account has sent.
+	///
+	/// Gets reset when the account gets reaped and is initializes to the current block number.
```

Comment:
```suggestion
	/// Gets reset when the account gets reaped.
```
Please don't make assumptions.